login-workflow/example2/src/actions/RegistrationUIActions.tsx

```diff
@@ -59,6 +59,15 @@ export const ProjectRegistrationUIActions: () => RegistrationUIActions = () => (
         return true;
     },
 
+    validateUserRegistrationRequest: async (validationCode: string, validationEmail?: string): Promise<boolean> => {
```

Comment:
Add param info like other functions.

for e.g.
`
/**
     * The user has tapped on an email link inviting them to register with the application.
     * The application should validate the code provided by the link.
     *
     * @param validationCode  Registration code provided from the link.
     * @param validationEmail  Email provided from the invitation email link (optional) `?email=addr%40domain.com`.
     *
     * @returns Resolves when the code is valid. True if registration is complete, False if account information is needed.
     *          If the code is not valid a rejection will occur with an error message.
     */
`